reconcile/utils/terrascript_aws_client.py

```diff
@@ -2512,6 +2512,45 @@ def populate_tf_resource_role(self, spec):
 
         self.add_resources(account, tf_resources)
 
+    def populate_saml_iam_role(
+        self,
+        account: str,
+        name: str,
+        saml_provider_name: str,
+        policies: list[str],
+        max_session_duration_hours: int = 1,
+    ) -> None:
+        """Manage the an IAM role needed for SAML authentication."""
+        managed_policy_arns = [
+            f"arn:{self._get_partition(account)}:" + f"iam::aws:policy/{policy}"
+            for policy in policies
```

Comment:
Will there be any duplicate `policy` in the list, will it cause error if there is duplicate `arn`?

Reply:
Good point! I've added a check.